source/game.broadcastmaterial.programme.bmx

```diff
@@ -347,6 +347,23 @@ Type TProgramme Extends TBroadcastMaterialDefaultImpl {_exposeToLua="selected"}
 		Return data.GetQuality()
 	End Method
 
+	'override
+	'reduce movie quality if not sent with maximal topicality
+	Method GetTopicalityModifier:Float()
+		Local topicality:Float = data.GetTopicality()
+		Local diff:Float = data.maxTopicalityCache - topicality
```

Comment:
Hat es einen Grund, dass 
- "topicality" statt direkt "data.GetTopicality()"
- und "data.maxTopicalityCache" (kann veraltet sein) statt "data.GetMaxTopicality()" aufgerufen wird?

Reply:
In Zeile 353 hole ich ja mit GetTopicality() den aktuellen Wert. Dort wird via GetMaxTopicality() der Cache aktualisiert. Ein etwas unsauberer Versuch überflüssige Berechnungen zu vermeiden.